py3status/modules/mpd_status.py

```diff
@@ -6,6 +6,10 @@
     cache_timeout: how often we refresh this module in seconds (default 2)
     format: template string (see below)
         (default '%state% [[[%artist%] - %title%]|[%file%]]')
+    format_mpd_not_authenticating: string to print
+        (default 'mpd_status: Authentication failed')
+    format_mpd_not_running: string to print
+        (default 'mpd_status: Connection refused')
```

Comment:
I know why you added these but I'm opposed to using format_... for this.  Currently we don't have a good solution.

Options could be `error_...` or my preference is that they are just fixed but translated.  These are errors more than things to display.  The max_width should just be fixed on it's own imho